jbot/src/main/java/me/ramswaroop/jbot/core/slack/models/Event.java

```diff
@@ -62,6 +65,8 @@
     private String domain;
     @JsonProperty("email_domain")
     private String emailDomain;
+    @JsonProperty("team")
+    private String teamId;
```

Comment:
So you're saying team_id is in all the responses except when subtype is pinned_item, unpinned_item, message_changed and message_deleted. Am I right?